stdlib/bytesLabels.mli

```diff
@@ -302,6 +302,22 @@ val equal: t -> t -> bool
 (** The equality function for byte sequences.
     @since 4.05.0 *)
 
+val to_hex : bytes -> bytes
+(** Return the printable hexadecimal representation of the given byte sequence.
```

Comment:
Please mention the encoding and the casing of letters. [Here's](https://erratique.ch/software/b0/doc/B00_std/String/Ascii/index.html#val-to_hex) a doc string you may want to copy cat.

Reply:
I'd just moved the comment from `Digest.to_hex` - I've updated them all (including the original), thanks.